src/index.tsx

```diff
@@ -26,12 +26,18 @@ import { loadGETheme } from './themes';
 import { Mode } from './types/action';
 import { IHistoryItem } from './types/history';
 
+// removes the loading spinner from GE html after the app is loaded
 const spinner = document.getElementById('spinner');
-
 if (spinner !== null) {
   (spinner as any).parentElement.removeChild(spinner);
 }
 
+// removes the loading spinner from the portal team html after GE loads
+const apiExplorer = document.getElementsByTagName('api-explorer')[0];
+if (apiExplorer && apiExplorer !== null) {
```

Comment:
Why do you have to check explicitly that apiExplorer is not equal to null?

Reply:
I don't.. Was being super safe

Reply:
It's unnecessary, `if (apiExplorer) {..}` should be enough.